src/components/ReleaseService/ReleasePlan/ReleasePlanListRow.tsx

```diff
@@ -25,9 +31,9 @@ const ReleasePlanListRow: React.FC<React.PropsWithChildren<RowFunctionArgs<Relea
             workspaceName: namespace,
             applicationName: obj.spec.application,
           })}
-          title={obj.spec.application}
+          title={obj.application?.spec?.displayName ?? obj.spec.application}
         >
-          {obj.spec.application}
+          {obj.application?.spec?.displayName ?? obj.spec.application}
```

Comment:
create a utility to get the application display name

```
const getApplicationDisplayName = (application) => application.spec.displayName ?? application.metadata.name;
```

Reply:
updated